src/main/java/pl/iordervivi/data/scrapper/gov/covid19/service/SickInRegionsDtoService.java

```diff
@@ -0,0 +1,19 @@
+package pl.iordervivi.data.scrapper.gov.covid19.service;
+
+import org.springframework.stereotype.Service;
+import pl.iordervivi.data.scrapper.gov.covid19.dto.SickInRegionsDto;
+
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class SickInRegionsDtoService {
```

Comment:
serwis do klas DTO? to mi wygląda jak zwykła metoda pomocnicza która powinna być częścią `DataProviderImpl`

Reply:
przerzuciłem metodę